src/test/scala/org/broadinstitute/hail/driver/FilterAllelesSuite.scala

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.hail.driver
```

Comment:
I might put this in the `methods` package -- that's where the other filter stuff is at the moment.